server/jetstream_cluster.go

```diff
@@ -158,6 +162,7 @@ const (
 	defaultStoreDirName  = "_js_"
 	defaultMetaGroupName = "_meta_"
 	defaultMetaFSBlkSize = 1024 * 1024
+	jsPeerExcludeTag     = "!jetstream"
```

Comment:
Suggest `jsExcludePlacement` value string, case-insensitive, as clearer meaning.

Reply:
So we want JS to stay active while we evacuate any assets etc but just have it not applicable for placement correct?

Reply:
Correct.  A marker on a server so it is not considered for new placements.